cypress/integration/conditions/alphanumeric_conditions.cy.js

```diff
@@ -244,6 +245,23 @@ describe('alphanumeric types', () => {
                     operator
                 )
             })
+            cy.getBySel('alphanumeric-condition-type')
+                .closest('[data-test=dhis2-uicore-layer]')
```

Comment:
There are 4 places where this data-test id is used in this manner. Is it possible to make a custom chainable cy command to do this? e.g. `closeCoreLayer`

<img width="823" alt="image" src="https://user-images.githubusercontent.com/6113918/184310917-412b6f91-3305-40f2-8f36-9680b85b86b6.png">

Reply:
Either that or we could at least extract that into a `closePopper` function to only do it once. I'll refactor it in my next PR